OPHD/main.cpp

```diff
@@ -70,7 +70,8 @@ int main(int /*argc*/, char *argv[])
 	try
 	{
 		Filesystem& f = Utility<Filesystem>::init<Filesystem>(argv[0], "OutpostHD", "LairWorks");
-		f.mount("data");
+		// Use executable's absolute directory in case environment path is not pointing to expected directory
+		f.mount(f.basePath() + "data");
```

Comment:
I made this change earlier, and it was undone by Leeor in:
https://github.com/OutpostUniverse/OPHD/commit/76014230ae0c2dd1027193030796ece38db05627

Might want to discuss with him about why. I'm not sure I entirely understand why it was reverted.